.dependabot/config.yml

```diff
@@ -7,3 +7,6 @@ update_configs:
       - match:
           dependency_name: "@date-io/luxon"
           version_requirement: "<=2.4.0"
+      - match:
```

Comment:
I wonder for each ignored_update, if we should get in the practice of adding a comment with a link to the PR that describes why the entry exists (for future reference).

i.e. Maybe simply include above each match line:
`# see https://github.com/target/goalert/pull/634 `

What do you think?